htmLawed.php

```diff
@@ -519,7 +519,7 @@ function hl_tag($t){
    $v = str_replace("­", ' ', (strpos($v, '&') !== false ? str_replace(array('&#xad;', '&#173;', '&shy;'), ' ', $v) : $v)); # double-quoted char: soft-hyphen; appears here as "­" or hyphen or something else depending on viewing software
    if($k == 'srcset'){
     $v2 = '';
-    $pattern = "/(?:[^\"'\s]+\s*(?:\d+[wx])+)/";
+    $pattern = "/(?:[^\"'\s]+\s*(?:\d+(?:\.\d+)?[wx])+)/";
```

Comment:
According to [spec](https://html.spec.whatwg.org/multipage/images.html#srcset-attribute), we might want `\d+w|\d+(?:\.\d+)?x` but maybe this simple regex will do.

Reply:
That's a good point, fixed. Thanks